test/data/corner_cases.out.mskcc.txt

```diff
@@ -1,5 +1,5 @@
 #genome_nexus_version: 0-unknown-version-SNAPSHOT
 #isoform: mskcc
-Hugo_Symbol	Entrez_Gene_Id	Center	NCBI_Build	Chromosome	Start_Position	End_Position	Strand	Consequence	Variant_Classification	Variant_Type	Reference_Allele	Tumor_Seq_Allele1	Tumor_Seq_Allele2	dbSNP_RS	dbSNP_Val_Status	Tumor_Sample_Barcode	Matched_Norm_Sample_Barcode	Match_Norm_Seq_Allele1	Match_Norm_Seq_Allele2	Tumor_Validation_Allele1	Tumor_Validation_Allele2	Match_Norm_Validation_Allele1	Match_Norm_Validation_Allele2	Verification_Status	Validation_Status	Mutation_Status	Sequencing_Phase	Sequence_Source	Validation_Method	Score	BAM_File	Sequencer	t_ref_count	t_alt_count	n_ref_count	n_alt_count	HGVSc	HGVSp	HGVSp_Short	Transcript_ID	RefSeq	Protein_position	Codons	Exon_Number	Annotation_Status
-MET	4233		GRCh37	7	116411872	116411900	+	splice_region_variant,intron_variant	Splice_Region	DEL	TAACAAGCTCTTTCTTTCTCTCTGTTTTA	-	-																								ENST00000397752.3:c.2888-31_2888-3del		p.X963_splice	ENST00000397752	NM_000245.2	963			SUCCESS
-PCM1	5108		GRCh37	8	17796382	17796383	+	missense_variant	Missense_Mutation	DNP	AC	GT	GT	rs754721723																							ENST00000325083.8:c.476_477inv	p.Asn159Ser	p.N159S	ENST00000325083	NM_006197.3	159	aAC/aGT	5/39	SUCCESS
+Hugo_Symbol	Entrez_Gene_Id	Center	NCBI_Build	Chromosome	Start_Position	End_Position	Strand	Consequence	Variant_Classification	Variant_Type	Reference_Allele	Tumor_Seq_Allele1	Tumor_Seq_Allele2	dbSNP_RS	dbSNP_Val_Status	Tumor_Sample_Barcode	Matched_Norm_Sample_Barcode	Match_Norm_Seq_Allele1	Match_Norm_Seq_Allele2	Tumor_Validation_Allele1	Tumor_Validation_Allele2	Match_Norm_Validation_Allele1	Match_Norm_Validation_Allele2	Verification_Status	Validation_Status	Mutation_Status	Sequencing_Phase	Sequence_Source	Validation_Method	Score	BAM_File	Sequencer	t_ref_count	t_alt_count	n_ref_count	n_alt_count	IGNORE_Genome_Nexus_Original_Chromosome	IGNORE_Genome_Nexus_Original_Start_Position	IGNORE_Genome_Nexus_Original_End_Position	IGNORE_Genome_Nexus_Original_Reference_Allele	IGNORE_Genome_Nexus_Original_Tumor_Seq_Allele1	IGNORE_Genome_Nexus_Original_Tumor_Seq_Allele2	HGVSc	HGVSp	HGVSp_Short	Transcript_ID	RefSeq	Protein_position	Codons	Exon_Number	Annotation_Status
```

Comment:
Not sure if we want to add those `IGNORE_Genome_Nexus_Original_` columns in example data?